src/lib/Fullscreen.js

```diff
@@ -73,33 +67,21 @@ class Fullscreen extends EventEmitter {
      * @return {boolean} In fullscreen or not
      */
     isFullscreen(element) {
-        let fullscreen;
         if (this.isSupported()) {
-            fullscreen = !!(
-                document.fullscreenElement ||
-                document.mozFullScreenElement ||
-                document.webkitFullscreenElement ||
-                document.msFullscreenElement
-            );
-        } else {
-            fullscreen = element instanceof HTMLElement && element.classList.contains(CLASS_FULLSCREEN);
+            return !!fscreen.fullscreenElement;
         }
-        return fullscreen;
+
+        return element instanceof HTMLElement && element.classList.contains(CLASS_FULLSCREEN);
     }
 
     /**
      * Fires events when the fullscreen state changes
      *
      * @private
-     * @param {HTMLElement|Event} el - Fullscreen element
      * @return {void}
      */
-    fullscreenchangeHandler = (el) => {
-        let enter = false;
-
-        enter = (this.isSupported() && this.isFullscreen()) || (!this.isSupported() && !this.isFullscreen(el));
-
-        if (enter) {
+    fullscreenchangeHandler = () => {
+        if (this.isFullscreen()) {
```

Comment:
Before it was checking `isFullscreen(el)`. This for sure isn't needed anymore?

Reply:
Correct. This method is only called as a handler for fscreen's `fullscreenchange` event, which gets its element from `fscreen.fullscreenElement`.